vnext/Desktop/WebSocketResourceFactory.cpp

```diff
@@ -15,11 +15,10 @@ namespace Microsoft::React {
 #pragma region IWebSocketResource static members
 
 /*static*/
-shared_ptr<IWebSocketResource>
-IWebSocketResource::Make(const string &urlString, bool legacyImplementation, bool acceptSelfSigned) {
-  if (GetFeatureGate("UseWinRTWebSocket")) {
+shared_ptr<IWebSocketResource> IWebSocketResource::Make(const string &urlString) {
+  if (!GetFeatureGate("UseBeastWebSocket")) {
```

Comment:
>! [](start = 6, length = 1)

nit: flip the condition and if/else?

Reply:
The feature gates are transient and prone to eventual deletion.
Not sure we should structure the code around them.